geti_sdk/data_models/algorithms.py

```diff
@@ -20,7 +20,7 @@
 from geti_sdk.data_models.utils import str_to_optional_enum_converter
 
 
-@attr.s(auto_attribs=True)
+@attr.define(auto_attribs=True)
```

Comment:
I think with the new api `auto_attribs` should be auto-detected, so it shouldn't be necessary to pass it explicitly